### ISSUE TITLE
Product not showing up when you access them the second time

### ISSUE THREAD

**Issue by shagilsiddiqui** [inspireui/support#4311]
Hi,
I am using FluxStore for Shopify
When I click on a category I get product list the first time when I refresh or click on it again on the same category nothing displays, and the same for any other category but only once as well.
Flutter
version: 1.7.5+3001
![86539793-688f0d00-becd-11ea-8c64-5c454d8b19f9](https://user-images.githubusercontent.com/40637140/88151946-f59dca00-cc20-11ea-99a4-1d17ee888ab7.jpg)

**Comment by luyx2412** [comment]
Hi @shagilsiddiqui  I just invited you to fluxshopify latest repo, please merge with your code which fixed the issue.

Thanks

**Comment by luyx2412** [comment]
Closed ticket because no longer response.

P/S: Feel free to create new ticket issue.
